Tests/XExtensionItemParametersTests.m

```diff
@@ -75,6 +75,16 @@ - (void)testSourceApplication {
     XCTAssertEqualObjects(inputParams.sourceApplication, outputParams.sourceApplication);
 }
 
+- (void)testSourceApplicationFromBundle {
+    XExtensionItemMutableParameters *inputParams = [[XExtensionItemMutableParameters alloc] init];
+    inputParams.sourceApplication = [[XExtensionItemSourceApplication alloc] initWithAppNameFromBundle:[NSBundle bundleForClass:[self class]]
```

Comment:
`[NSBundle mainBundle].infoDictionary` is empty, whereas `[NSBundle bundleForClass:[self class]].infoDictionary` returns the Test target’s `Info.plist` values.